src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigratePackageDependenciesAndToolsRule.cs

```diff
@@ -69,17 +69,21 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
                     migrationRuleInputs.ProjectXproj); 
             }
 
-            // Tools
             MigrateTools(project, migrationRuleInputs.OutputMSBuildProject);
         }
 
-        private void MigrateImports(ProjectItemGroupElement commonItemGroup, TargetFrameworkInformation targetFramework)
+        private void MigrateImports(ProjectPropertyGroupElement commonPropertyGroup, TargetFrameworkInformation targetFramework)
         {
             var transform = ImportsTransformation.Transform(targetFramework);
+
             if (transform != null)
             {
                 transform.Condition = targetFramework.FrameworkName.GetMSBuildCondition();
-                _transformApplicator.Execute(transform, commonItemGroup);
+                _transformApplicator.Execute(transform, commonPropertyGroup);
+            }
+            else
+            {
+                MigrationTrace.Instance.WriteLine($"{nameof(MigratePackageDependenciesAndToolsRule)}: imports transform null for {string.Join(";", targetFramework.FrameworkName.GetShortFolderName())}");
```

Comment:
Why the `string.Join` here?  Isn't `targetFramework.FrameworkName.GetShortFolderName()` a single string?